mavis/test/data/__init__.py

```diff
@@ -153,6 +155,10 @@ def create_file_from_template(
         if self.nurse:
             static_replacements["<<NURSE_EMAIL>>"] = self.nurse.username
 
+        if self.clinic:
+            static_replacements["<<CLINIC_LOWER>>"] = self.clinic[0].name.lower()
+            static_replacements["<<CLINIC>>"] = self.clinic[0].name
```

Comment:
To prepare for any future tests that might rely on multiple clinics, could we use the same approach we've got schools where we have an index in the name? Something like:

```
CLINIC_0_NAME
CLINIC_0_NAME_LOWER
```